README.md

```diff
@@ -35,13 +35,15 @@ so you will need to install golang and set the following value in
 
 Alternately you can set `openssl` which has no other dependencies.
 
-You now need to start the following three services.
+You now need to start the following services.
 
 `# keylime_verifier`
 
 `# keylime_registrar`
 
-`# keylime_agent`
+To run the agent, navigate to the rust-keylime directory and start the agent. 
+
+`# RUST_LOG=keylime_agent=trace cargo run `
```

Comment:
This needs to be updated in our main documentation. I would use 
`# RUST_LOG=keylime_agent=trace cargo run --bin keylime_agent`